stand/defs.mk

```diff
@@ -128,6 +128,11 @@ SSP_CFLAGS=
 CFLAGS+=	-ffreestanding ${CFLAGS_NO_SIMD}
 .if ${MACHINE_CPUARCH} == "aarch64"
 CFLAGS+=	-mgeneral-regs-only -ffixed-x18 -fPIC
+# Disable Morello support in the bootloader. EFI needs to understand
+# capabilities before we can enable this as it may have interrupts enabled
+# so will need to save/restore the full capability registers
```

Comment:
Needs an XXX

Reply:
Yeah I guess so. Although searching for XXX is not very helpful:
```
git grep -E 'XXX\w*:' | wc -l
    3866
```
I guess we can search for `XXX.*:.*morello`

Reply:
It's better than nothing, and also draws attention to itself when skimming the file, at least with my syntax highlighting.